File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/Application.java
package com.antonio.samir.meteoritelandingsspots;


import android.content.Context;

import com.antonio.samir.meteoritelandingsspots.util.analytics.AnalyticsUtil;

public class Application extends android.app.Application {

    private static Context applicationContext;

    public static Context getContext() {
        return applicationContext;
    }

    public void onCreate() {
        super.onCreate();

        applicationContext = getApplicationContext();

        AnalyticsUtil.start(this);

    }
}

File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/Application.kt
package com.antonio.samir.meteoritelandingsspots

import com.antonio.samir.meteoritelandingsspots.di.appModules
import org.koin.android.ext.koin.androidContext
import org.koin.core.context.startKoin

open class Application : android.app.Application() {

    override fun onCreate() {
        super.onCreate()
        // start Koin!
        startKoin {
            // declare used Android context
            androidContext(this@Application)
            // declare modules
            modules(appModules)
        }
    }

}
